### PR TITLE
INT1 Remove sort filter Page Library

### PR DESCRIPTION
**Pull request by jlee-enquizit**
Here we fix the search in Pages Library:
now when you clear the search terms, the page list resets

### CODE REVIEW

File: apps/modernization-ui/src/apps/page-builder/components/TableMenu/TableMenu.tsx [hunk 1]
```diff
@@ -34,6 +34,12 @@ export const TableMenu = ({ tableType, searchQuery, setSearchQuery, onDownloadIc
         }
     }, [searchQuery]);
 
+    useEffect(() => {
+        if (!keywords || keywords === '') {
+            setSearchQuery('');
+        }
+    }, [keywords]);
```
Comment (adamloup-enquizit) [inline apps/modernization-ui/src/apps/page-builder/components/TableMenu/TableMenu.tsx:41]: question: Should this case be handled in the on-click?  it's changing search query without having to hit submit, not sure if that is desired behavior or not.

**Comment by mpeels** [comment]
> Here we fix the search in Pages Library: now when you clear the search terms, the page list resets

PR is missing 
1. Link to Jira ticket
2. Steps to verify

**Comment by jlee-enquizit** [comment]
[CNFT2-1336](https://cdc-nbs.atlassian.net/browse/CNFT2-1336)

File: apps/modernization-ui/src/apps/page-builder/components/TableMenu/TableMenu.tsx (hunk 1 above)
Comment (jlee-enquizit) [reply apps/modernization-ui/src/apps/page-builder/components/TableMenu/TableMenu.tsx:41]: it's setting the query, but it won't search until 'Search' is clicke.
It searches for '' when the user clicks 'X' to clear the searchQuery, search is run automatically only then.

**Comment by mpeels** [comment]
Should the `onClick` for the `X` just re-run the search?

**Comment by jlee-enquizit** [comment]
> Should the `onClick` for the `X` just re-run the search?

The X is inserted via a prop, we don't have access to add an `onClick` function to it. 
We had to find another way to run it which, in this case, is when the search field is empty.

**Review by adamloup-enquizit — APPROVED** [review]

**Pull request merged**